site/ba.njk

```diff
@@ -0,0 +1,151 @@
+---
+layout: default
+meta_title: Pokret za zaustavljanje COVID-19 pandemije
+meta_description: Pridonesi borbi i spasi živote ne radeći nista! Jednostavno ostani jebeno kod kuće!
+body_class: home
+title: '#StayTheFuckHome'
+lang: en
```

Comment:
Wrong language code, "bs" for Bosnian. Dtto for file name and site/globals/langs.json.